aiomcache/client.py

```diff
@@ -9,13 +10,18 @@
 __all__ = ['Client']
 
 _T = TypeVar("_T")
-_Result = Tuple[Dict[bytes, bytes], Dict[bytes, Optional[int]]]
+_FlagT = TypeVar("_FlagT")
+_ClientT = Union[bytes, _FlagT]
+_Result = Tuple[Dict[bytes, _ClientT], Dict[bytes, Optional[int]]]
+
+_get_flag_callable = Callable[[bytes, int], Awaitable[_FlagT]]
+_set_flag_callable = Callable[[_FlagT], Awaitable[Tuple[bytes, int]]]
 
 
 def acquire(func: Callable[..., Awaitable[_T]]) -> Callable[..., Awaitable[_T]]:
 
     @functools.wraps(func)
-    async def wrapper(self: "Client", *args: object, **kwargs: object) -> _T:
+    async def wrapper(self: "Client", *args: Tuple[Any], **kwargs: Dict[str, Any]) -> _T:
```

Comment:
This was correct before. The change says that every positional argument must be a tuple and even kwarg must be a dict.

Reply:
reverted